src/jazkarta/easyformplugin/savecontent/browser.py

```diff
@@ -16,3 +21,34 @@ def has_saved_content(self):
 
     def saved_content_url(self):
         return self.context.absolute_url() + '/' + STORAGE_ID
+
+
+class CSVDownload(BrowserView):
+
+    def __call__(self):
+        items_dicts = []
+        keys = OrderedDict()
+        for obj in self.context.objectValues():
+            behavior_assignment = EasyformSchemaBehaviorAssignment(obj)
```

Comment:
I think you only need to do this once, and can just use `behavior_assignment.interface`.